packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -70,6 +82,10 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
       glossValue ?? ''
     );
     const hasMachineSuggestion = !gloss && !suggestions[0] && !!machineGloss;
+    const notesQuery = useQuery(
+      ['verse-translator-notes', targetLanguage?.code ?? '', verseId],
+      () => apiClient.verses.findNotes(verseId, targetLanguage?.code ?? '')
+    );
```

Comment:
I'm just realizing that perhaps we should lift this to the view and pass a `hasNote` prop so that we only try to load it once and not for each word

Reply:
Done.